projects/js-packages/components/components/threat-modal/threat-technical-details.tsx

```diff
@@ -17,6 +17,11 @@ const ThreatTechnicalDetails = (): JSX.Element => {
 
 	const [ open, setOpen ] = useState( false );
 
+	let toggleContent = __( 'Show the technical details', 'jetpack-components' );
+	if ( open ) {
+		toggleContent = __( 'Hide the technical details', 'jetpack-components' );
+	}
```

Comment:
Setting it up this way instead of using a ternary because:
- Supplying a dummy arg to avoid bad minification breaks the `js-packages/components` build
- Not supply that dummy arg breaks the `plugins/protect` build